example/rnn/bucketing/cudnn_rnn_bucketing.py

```diff
@@ -67,6 +68,10 @@
                     help='dropout probability (1.0 - keep probability)')
 parser.add_argument('--rnntype', type=str, default='lstm',
                     help='rnn type: gru, lstm, rnn_tanh and rnn_relu are supported')
+parser.add_argument('--dtype', type=str, default='float32',
+                    help='if float16 is provided AMP convert model'
+                         'is used to convert model to mixed precision model'
+                         'before running inference')
```

Comment:
what are the supported values for dtype? only FP16?

Reply:
maybe we should define the possible values in the hlep message [float16, float32] so that its clear we dont support int8 yet?

Reply:
sure! i have updated the help